### PR TITLE
Create List screen

### PR DESCRIPTION
**Pull request by andreasalstrup**
Solves:
#6 
#11

### CODE REVIEW

**Review by mikejensen0 — APPROVED** [review]
WOMM LGTM++

File: app/(tabs)/index.tsx [hunk 1]
```diff
@@ -1,78 +1,135 @@
...
+import { FlatList } from 'react-native-gesture-handler';
+import { FontAwesome5 } from '@expo/vector-icons';
+import Swipeable from 'react-native-gesture-handler/Swipeable';
 
-import useGun from '../../hooks/useGun';
-const { gun, app, user, SEA } = useGun();
+const DATA = [
+  {
+      name: 'Bananas',
+      data: [{user: "Test bruger", date: "2020.11.01", price: 3}],
```
Comment (UnluckyBird) [inline app/(tabs)/index.tsx:11]: Inconsistent whitespace

File: app/(tabs)/index.tsx [hunk 2]
```diff
@@ -1,78 +1,135 @@
...
-    super(props);
-    this.state = {
-      text: 'Whats your name?',
-      name: '',
-      paste: '',
-    };
-  }
+function leftSwipeAction() {
+  return (
+    <View style={{backgroundColor: '#5CBCA9', justifyContent: 'center', alignItems: 'flex-end'}}>
```
Comment (UnluckyBird) [inline app/(tabs)/index.tsx:57]: Should probably be moved to stylesheet

File: app/(tabs)/index.tsx [hunk 3]
```diff
@@ -1,78 +1,135 @@
...
+        style={styles.swipe}
+        />
+    </View>
+  )
+}
 
-  $user = user.create('test', 'testtest', () => {})
+function rightSwipeAction() {
+  return (
+    <View style={{backgroundColor: '#E35F52', justifyContent: 'center', alignItems: 'flex-end'}}>
```
Comment (UnluckyBird) [inline app/(tabs)/index.tsx:70]: Same as above

File: app/(tabs)/index.tsx [hunk 4]
```diff
@@ -1,78 +1,135 @@
...
 
-  componentDidMount(): void {
-    this.$user;
-    app.on((data: any) => {
-      console.log('data', data);
-      this.setState({paste: data.paste});
-    });
+function swipeHandler(dir: 'left' | 'right') {
+  if (dir == "left") {
+      alert(dir);
```
Comment (UnluckyBird) [inline app/(tabs)/index.tsx:83]: The constant alerting is kinda annoying maybe just log to console until proper implementation is done

File: app/(tabs)/index.tsx [hunk 5]
```diff
@@ -1,78 +1,135 @@
...
 }
 
 const styles = StyleSheet.create({
   container: {
-    flex: 1,
-    alignItems: 'center',
-    justifyContent: 'center',
+      borderBottomColor: '#000000',
+      padding: 2,
+    },
```
Comment (UnluckyBird) [inline app/(tabs)/index.tsx:104]: Inconsistent whitespace

**Review by UnluckyBird — APPROVED** [review]
Some small suggestions

File: app/(tabs)/index.tsx (hunk 3 above)
Comment (andreasalstrup) [reply app/(tabs)/index.tsx]: Fixed

File: app/(tabs)/index.tsx (hunk 2 above)
Comment (andreasalstrup) [reply app/(tabs)/index.tsx]: Fixed

File: app/(tabs)/index.tsx (hunk 1 above)
Comment (andreasalstrup) [reply app/(tabs)/index.tsx]: Fixed

**Pull request merged**